_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -99,17 +99,18 @@ helm upgrade -i forseti forseti-security/forseti-security \
     --values=forseti-values.yaml
 ```
 
-### Deploying with config-validator on-GKE
+## Deploying with config-validator on-GKE
 
 The config-validator in Forseti on-GKE obtains policies from a policy-library in a Git repository via SSH.  The pre-requisites for this are as follows.
```

Comment:
I think this section should be updated to say that we support Git and GCS. The pre-requisites for Git can be moved into the Policy Library in a Git Repository section, and add the pre-reqs for GCS. I think the only pre-req for GCS is the user will need to create the `policy-library` directory in the server GCS bucket.